docs/general/ddl/data-types.rst

```diff
@@ -2351,13 +2351,19 @@ Example::
     ... ) VALUES (
     ...     'Alice in Wonderland',
     ...     {
-    ...         "age" = '10',
+    ...         "age" = '10'
     ...     }
     ... );
-    SQLParseException[line 8:5: no viable alternative at input 'VALUES (\n    'Alice in Wonderland',\n    {\n        "age" = '10',\n    }']
+    StrictDynamicMappingException[mapping set to strict, dynamic introduction of [age] within [protagonist] is not allowed]
+
+The insert above failed because the ``age`` sub-column is not defined.
+
+Sub-columns can be added to an ``OBJECT(STRICT)`` at a later stage:
+
+::
 
-The insert above failed because the ``protagonist`` column defines a ``name``
-column and does not define an ``age`` column.
+    cr> ALTER TABLE my_table ADD COLUMN protagonist['age'] SMALLINT;
+    ALTER OK, -1 rows affected  (0.056 sec)
```

Comment:
I've noticed that for all other queries on this page, the runtime is replaced with a placeholder (`(... sec)`).

Reply:
I am under the impression ... appears only when the statement uses more than one line.
Both `...` and `cr>` are inconvenient when you want to copy the SQL code to the clipboard, should I perhaps change this everywhere in the page so that we have code blocks with just SQL code and maybe separate blocks with just output?

Reply:
I was referring to this:
<img width="861" alt="Screenshot 2023-03-29 at 09 05 30" src="https://user-images.githubusercontent.com/218003/228453419-47a72d00-84ad-4f0a-8aa5-313f91fee605.png">

> Both ... and cr> are inconvenient when you want to copy the SQL code to the clipboard

When clicking the "copy" button on the top right corner of a code box, it (magically) filters out the prompt, output, etc, and correctly copies only the plain statement. But might be browser-dependent, does it work for you as well?

Reply:
those examples are btw run with automated doc test and therefore afaik need to be formatted that way ;)

Reply:
> should I perhaps change this everywhere in the page so that we have code blocks with just SQL code and maybe separate blocks with just output?

`cr>` is used as marker for the test framework to know that it's a code block that should be executed. The written output is then compared with the actual output from the execution to ensure the documentation stays up2date and that the examples actually work.

The duration in the output is replaced with `...` because it changes each time and would cause test failures.